### ISSUE TITLE
Highlight NEXT button so that users will know to click it

### ISSUE THREAD

**Issue by zoharbabin** [kaltura/kaltura-api-recipes#15]
Currently the NEXT button is often below fold, and that makes it hard to follow the path we want the user to take.
We need to somehow tell the user it's time to click the next button, and where it's located.

**Comment by rbren** [comment]
@devinllp any ideas here? I'm open to suggestions.

**Comment by zoharbabin** [comment]
@bobby-brennan  - let's think of this as part of the V2 work. Sounds like it could be solved better then.

**Comment by rbren** [comment]
sg. Closing
